api_v3/lib/KalturaErrors.php

```diff
@@ -582,6 +582,12 @@ class KalturaErrors extends APIErrors
 	 * Live reports Service
 	 */
 	const LIVE_REPORTS_WS_FAILURE = 'LIVE_REPORTS_WS_FAILURE;;failed to retrieve live analytics';
+	
+	/*
+	 * Analytics Service
+	 */
+	const ANALYTICS_QUERY_FAILURE = 'REPORTS_QUERY_FAILURE;ERROR_MSG;failed to retrieve analytics data - [@ERROR_MSG@]';
```

Comment:
Good point. Yes, it is certainly possible. I'll take it up with @kerkero to see how this fits in the work plan.